Makefile

```diff
@@ -18,6 +18,9 @@ deps:
 release:
 	./scripts/release.sh
 
+test:
+	go test -v ./pkg/...
+
```

Comment:
This would conflict with the `test` generated by `TARGETS := $(shell ls scripts | grep -v \\.sh)` expanding out `scripts/test` as a phony target.

Reply:
We're on a call making adjustments and changes will be pushed shortly for review.

Reply:
ah OK. You might mark it as WIP if you're going to iterate without wanting review on it.